v2/community/supertokens-core/self-hosted/api-keys.mdx

```diff
@@ -17,6 +17,8 @@ import TabItem from '@theme/TabItem';
 This is only relevant if you are running SuperTokens on your own.
 :::
 
+API keys are used to indentify authentication/authorization requests from a client trying to query a SuperTokens core instance. If the API key is not specified in the client's backend config, auth related APIs to the core will return a `401` `UNAUTHORIZED` error. 
```

Comment:
This is very very vague.. who is the client, where is the client's backend config?
Say something like this "API keys are used to authenticate requests made from the backend SDK to the SuperTokens core. By default, there is no API key that is required, but in case you add one in the core's config or are using the managed service, then, you will have to provide the API key in your backend SDK code - else the core will throw a `401` error."

Reply:
Rewrite this properly.